docs/Coding/hacking.rst

```diff
@@ -591,11 +591,11 @@ With an isometric view, this looks like:
     |                            (nat_x, nat_y+2)
 
 
-Neither is particularly good for a global Map operation such as Map wrapping or conversions to or from Map
-indexes. Something better is needed.
+Neither is particularly good for a global map operation such as map wrapping or conversions to or from map
+indexes. something better is needed.
```

Comment:
Typo: "something" should be left capitalized, it begins a sentence

Reply:
Distinct GUI elements are supposed to be capitalized. Map View, Cities View, etc. Messages Widget, Chat Widget, etc. Production Tab, General Tab, etc. should all be capitalized.

Reply:
> Distinct GUI elements are supposed to be capitalized. Map View, Cities View, etc. Messages Widget, Chat Widget, etc. Production Tab, General Tab, etc. should all be capitalized.

Not per the guidelines

Reply:
Ah, it got excised in PR #1334 by accident. I didn't catch that during review.